deb/openmediavault/srv/salt/omv/deploy/chrony/default.sls

```diff
@@ -27,6 +27,7 @@ configure_chrony:
     - template: jinja
     - context:
         timeservers: "{{ config.ntp.timeservers }}"
+        clients: "{{ config.ntp.clients }}"
```

Comment:
Due the fact that we now pass multiple configuration options to the template, i suggest to use this.
```suggestion
    - context:
        ntp_config: "{{ config.ntp }}"
```

Reply:
I'm having a bit of trouble with this one. When I change the context, the Jinja 2 template fails on the references to timeservers and clients. I tried a few different ways of referencing the variables inside the template, but they've all failed. My Python is a little rusty - can you give me a hint as to the template change that is needed to go with this?

Reply:
oh right, it must be

```
    - context:
        ntp_config: {{ config.ntp | json }}
```
You need to replace `timeservers` and `clients` with `ntp_config.timeservers` and `ntp_config.clients` in the `etc-chrony-chrony_conf.j2` file.

Reply:
This fails with

```
Rendering SLS 'base:omv.deploy.chrony.default' failed: while parsing a block mapping in "<unicode string>", line 29, column 9 did not find expected key in "<unicode string>", line 29, column 24
```

I'll investigate. My lack of experience with Python is showing. :grimacing:

Reply:
I got it. Had quotes around `{{ config.ntp | json }}` leftover from before that was throwing everything off.